intents/retake/_schema.json

```diff
@@ -0,0 +1,230 @@
+{
+    "intents": [
+        {
+            "name": "RetakeClassIntent",
+            "slots": [
+                {
+                    "name": "CourseAbbrev",
```

Comment:
should be broken up into two slots, subject and number. number should use AMAZON.Numbers and subject should be global for all applications.